src/Blog_App_frontend/src/AddBlog.css

```diff
@@ -1,13 +1,13 @@
-/* AddBlog.css */
+
 
 .add-blog-container {
-  max-width: 1200px; /* Increase the max-width */
+  max-width: 1200px; 
   margin: 0 auto;
   background: #fff;
-  padding: 40px; /* Increase padding for more space */
+  padding: 40px;
   border-radius: 10px;
   box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
-  margin-top: 40px; /* Increase margin-top for more space */
+  margin-top: 40px;
```

Comment:
instead of
```
margin:0 auto;
margin-top: 40px;
```
you can do

```
margin: 40px auto 0;
```
also generally its better to use em for spacing than px

Reply:
ok